bootstrap.sh

```diff
@@ -19,39 +46,89 @@ then
-sed -i.template "s|Country-Template|$COUNTRY_NAME|g" README.md setup.py .github/workflows/workflow.yml .github/PULL_REQUEST_TEMPLATE.md CONTRIBUTING.md $all_module_files
+sed -i.template "s|openfisca-country_template|openfisca-$NO_SPACES_JURISDICTION_LABEL|g" README.md Makefile pyproject.toml CONTRIBUTING.md
+sed -i.template "s|country_template|$CODE_JURISDICTION_LABEL|g" README.md pyproject.toml .flake8 .github/workflows/workflow.yml Makefile MANIFEST.in $all_module_files
+sed -i.template "s|Country-Template|$JURISDICTION_NAME|g" README.md pyproject.toml .github/workflows/workflow.yml .github/PULL_REQUEST_TEMPLATE.md CONTRIBUTING.md
 sed -i.template -e "3,${last_bootstrapping_line_number}d" README.md  # remove instructions lines
-sed -i.template "s|country-template|$lowercase_country_name|g" README.md
-sed -i.template "s|https://github.com/openfisca/openfisca-country-template|$URL|g" setup.py
+sed -i.template "s|https://example.com/repository|$REPOSITORY_URL|g" README.md CONTRIBUTING.md
+sed -i.template -e "1,${last_changelog_number}d" CHANGELOG.md  # remove country-template CHANGELOG leaving example
+sed -i.template "s|https://github.com/openfisca/country-template|$REPOSITORY_URL|g" pyproject.toml
+sed -i.template "s|repository_folder|$REPOSITORY_FOLDER|g" README.md
 find . -name "*.template" -type f -delete
 
 set +x
 
-git mv openfisca_country_template openfisca_$lowercase_country_name
+git mv openfisca_country_template openfisca_$CODE_JURISDICTION_LABEL
 
 git rm bootstrap.sh
 git add .
-git commit --no-gpg-sign --message 'Customise country-template through script' --author='OpenFisca Bot <bot@openfisca.org>'
+git commit --no-gpg-sign --message "$second_commit_message" --author='OpenFisca Bot <bot@openfisca.org>'
+echo -e "${YELLOW}--\033[0m"
+echo -e "${PURPLE}Second git commit made: \033[0m${BLUE}$second_commit_message\033[0m"
 git remote add origin $REPOSITORY_URL.git
+echo -e "${PURPLE}Git remote origin added: \033[0m${BLUE}$REPOSITORY_URL.git\033[0m"
+echo -e "${YELLOW}--\033[0m"
 
-echo '************'
-echo '* All set! *'
-echo '************'
+echo -e "${YELLOW}***********************************************************************\033[0m"
+echo -e "${YELLOW}*\033[0m  Navigate to your project folder via: "
+echo -e "${YELLOW}*\033[0m  $ ${BLUE}cd ../openfisca-$NO_SPACES_JURISDICTION_LABEL\033[0m"
+echo -e "${YELLOW}*\033[0m"
+echo -e "${YELLOW}*\033[0m  Setup, activate a virtual environment:"
+echo -e "${YELLOW}*\033[0m  $ ${BLUE}python3.9 -m venv .venv\033[0m"
+echo -e "${YELLOW}*\033[0m  $ ${BLUE}source .venv/bin/activate\033[0m"
+echo -e "${YELLOW}*\033[0m"
+echo -e "${YELLOW}*\033[0m  Ensure pip, build & twine are updated, then install project:"
+echo -e "${YELLOW}*\033[0m  $ ${BLUE}pip install --upgrade pip build twine\033[0m"
+echo -e "${YELLOW}*\033[0m  $ ${BLUE}pip install --editable .[dev] --upgrade\033[0m"
```

Comment:
I see the value of giving these instructions here, but I'm a bit concerned with the maintainability of these duplicate points from the README… Could we maybe find a smart way to `cat` the relevant part from the README? 🙂

Reply:
Have removed the indicated instructions by making more of a distinction between the one off bootstrap process and repeatable installation process in commit ([600f986](https://github.com/openfisca/country-template/pull/139/commits/600f9869bde7cdd00918f769c244b5b0d81136ae)).